docs/doc_tutorial/advanced/training_seo.rst

```diff
@@ -0,0 +1,135 @@
+---
+SEO
+---
+
+ViUR offers the possibility to make URLs SEO-friendly in multiple languages.
+For this you can define seo-identifier for each url component.
+
+
+Module
+------
+The modulename is the first part of an URL.
+SEO-Identfiers has to be set as class-attribute ``seoLanguageMap`` of type ``dict[str, str]`` in the module.
+It maps a *language* to the acording *identifier*.
+
+.. code-block:: python
+    :name: module seo-map
+    :caption: modules/myorders.py
+    :emphasize-lines: 4-7
+
+    from viur.core.prototypes import List
+
+    class MyOrders(List):
+        seoLanguageMap = {
+            "de": "bestellungen",
+            "en": "orders",
+        }
+
+By default the module would be available under */myorders*, the lowercase modulename.
+With the defined :attr:`seoLanguageMap`, it's now available under */de/bestellungen* and */en/orders*.
+
+Great, this part is now user and robot friendly :)
+
+
+Method
+------
+The methodname is usually the second part of an URL.
```

Comment:
```suggestion
The methodname is usually the second part of a URL.
```
and because I know people will fight me on thtis: https://www.techtarget.com/whatis/feature/Which-is-correct-a-URL-or-an-URL#:~:text=Although%20the%20U%20at%20the,use%20a%20rather%20than%20an.